app/mashup/Preload.scala

```diff
@@ -79,7 +80,11 @@ final class Preload(
       )
       .zip((ctx.userId.so(playbanApi.currentBan)).mon(_.lobby.segment("playban")))
       .zip(ctx.blind.so(ctx.me).so(roundProxy.urgentGames))
-      .zip(lastPostsCache.get {})
+      .zip((lastPostsCache.get {}).flatMap { posts =>
+        Future.sequence(posts.map(p => ctx.userId.so(relationApi.fetchBlocks(_, p.created.by)))).map {
+          blocks => (posts.zip(blocks)).filter(!_._2).map(_._1)
+        }
```

Comment:
this adds 15 database calls every time someone loads the homepage. 15 * 150 = 2250 new database requests per second

Reply:
@ornicar Updated to only do one database call. Incidentally it also makes the filtering logic simpler.

Reply:
Now it "only" adds 150 database requests per second. Still a stiff price to pay for a low value feature.

`relationApi.fetchBlocking` can return up to 500 usernames.